### PR TITLE
Update GCI Mentor Guidelines.

### PR DESCRIPTION
**Pull request by starkblaze01**
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts
* [X] PR is descriptively titled
* [X] PR body includes `fixes #0000`-style reference to original issue #3471 
* [X] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: doc/GCI_MENTOR_GUIDELINES.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ To be a GCI mentor at Public labs, there are some prerequisites that we expect y
 
 1. Solve a [fto issue](https://code.publiclab.org/). Provide the link to the first timer issue's pull request (PR) once it's merged.
 
-2. Solve a [help-wanted issue](https://github.com/publiclab/plots2/labels/help-wanted). Help wanted issues are issues which are not labelled as `gci-candidate` neither as `fto-candidate` nor as `first-timer-only`. Provide link to such issues' merged PR.
+2. Solve a [help-wanted issue](https://github.com/publiclab/plots2/labels/help-wanted). Help wanted issues are issues which are not labelled as `gci-candidate` neither as `fto` nor as `first-timer-only`. Provide link to such issues' merged PR.
```
Comment (SidharthBansal) [inline doc/GCI_MENTOR_GUIDELINES.md]: Can you please change this back to `fto-candidate`?

File: doc/GCI_MENTOR_GUIDELINES.md [hunk 2]
```diff
@@ -36,9 +38,21 @@ lines like
...
 
+**PR Review Guidelines:**
+
+1. PR should use 'Fixes #issue_no" in PR body.
+
+2. Code should be modular.
+
+3. Tests must be written if needed. All tests must pass before approval.
+
+4. Suggest code changes if any. Else approve the PR by commentting in the PR.
```
Comment (SidharthBansal) [inline doc/GCI_MENTOR_GUIDELINES.md]: Please take a look on https://github.com/publiclab/plots2/issues/3563 and add the remaining stuff.

Comment (SidharthBansal) [reply doc/GCI_MENTOR_GUIDELINES.md]: Also take a look on https://docs.google.com/document/d/1-BesjjJnQGAi5A9sS8bJoepmxRSmGZvZ9-uffcbhG_k/edit and add remaining stuff.

**Comment by starkblaze01** [comment]
I have made the necessary changes. Please review.

**Comment by SidharthBansal** [comment]
Great work!

**Comment by jywarren** [comment]
Hi -- this looks perfect; merging now. 👍 🎉  If you're looking for a new challenge, please take a look at https://code.publiclab.org#r=all; we prefer to leave `first-timers-only` issues for complete newcomers, but there are plenty of others we need help with! Thanks!

**Pull request merged**